src/domain/home/Home.tsx

```diff
@@ -33,13 +34,14 @@ const Home: FunctionComponent = () => {
     }
   };
 
+  useEffect(() => {
```

Comment:
I am not sure if this is the right approach to solve this: We get rid of the warning but could it be that we are just brushing the problem under the carpet?